src/responder/kcm/kcm_renew.c

```diff
@@ -385,6 +387,84 @@ int kcm_get_renewal_config(struct kcm_ctx *kctx,
     return ret;
 }
 
+static void kcm_renew_tgt(struct tevent_context *ev, struct tevent_timer *te,
+                      struct timeval current_time, void *private_data)
+{
+    struct auth_data *auth_data = talloc_get_type(private_data,
+                                                  struct auth_data);
+    struct tevent_req *req;
+}
+
+static void kcm_renew_tgt_done(struct tevent_req *req)
+{
+}
+
+errno_t kcm_renew_all_tgts(struct renew_tgt_ctx *renew_tgt_ctx)
+{
+    int ret;
+    hash_entry_t *entries;
+    unsigned long count;
+    size_t c;
+    time_t now;
+    struct auth_data *auth_data;
+    struct renew_data *renew_data;
+    struct tevent_timer *te = NULL;
+
+    ret = hash_entries(renew_tgt_ctx->tgt_table, &count, &entries);
+    if (ret != HASH_SUCCESS) {
+        DEBUG(SSSDBG_CRIT_FAILURE, "hash_entries failed.\n");
+        return ENOMEM;
+    }
+    DEBUG(SSSDBG_TRACE_INTERNAL, "Found [%lu] renewal entries.\n", count);
+
+    now = time(NULL);
+
+    for (c = 0; c < count; c++) {
+        renew_data = talloc_get_type(entries[c].value.ptr, struct renew_data);
+        DEBUG(SSSDBG_TRACE_INTERNAL,
+              "Checking [%s] for renewal at [%.24s].\n", renew_data->ccname,
+                  ctime(&renew_data->start_renew_at));
```

Comment:
Move the debug string to the first line, then you can fit it to two lines.

Reply:
Fixed.